source/includes/_event_notifications.md

```diff
@@ -69,6 +75,9 @@ The body of an event subscription will vary depending on the type of event being
             "id": "19",
             "type": "project"
           }
+        },
+        "message": {
+          "data": null
```

Comment:
interesting, do we ever expose that in Zapier? if not maybe we should remove it from the API history serializer?

Reply:
Hi @kknevitt. We don't - but can't guarantee no one with a direct integration to the api isn't using it. Happy to remove it?